lighthouse-core/test/results/sample_v2.json

```diff
@@ -3172,6 +3172,7 @@
     "disableStorageReset": false,
     "disableDeviceEmulation": false,
     "emulatedFormFactor": "mobile",
+    "channel": "cli",
```

Comment:
should it also be in the top level of the LHR? Maybe inside `environment`, next to `benchmarkIndex`, etc?

Reply:
Is there a benefit over using `lhr.configSettings.channel`?

Reply:
> Is there a benefit over using lhr.configSettings.channel?

because it's not really a setting, but now it kind of is, so that's fine